polaris.shopify.com/content/components/inline.md

```diff
@@ -32,5 +32,5 @@ examples:
 
 ## Related components
 
-- To create the large-scale structure of pages, [use the Columns](https://polaris.shopify.com/components/columns) and [Tile component](https://polaris.shopify.com/components/tile)
-- To display elements vertically, [use AlphaStack](https://polaris.shopify.com/components/alphastack)
+- To create the large-scale structure of pages, [use the Columns component](https://polaris.shopify.com/components/columns)
```

Comment:
```suggestion
```

Should we just remove this? I think we had a convo somewhere else that the point of "used for large structure" might be confusing.